azkaban-common/src/main/java/azkaban/jobExecutor/ProcessJob.java

```diff
@@ -26,6 +26,7 @@
 import azkaban.utils.Pair;
 import azkaban.utils.Props;
 import azkaban.utils.SystemMemoryInfo;
+import com.sun.xml.internal.fastinfoset.util.StringArray;
```

Comment:
Intellij added this for me, and I didn't validate if it was needed. I went ahead and removed this import. Thanks for pointing this out.